ciphers/hill_cipher.py

```diff
@@ -129,32 +166,55 @@ def make_decrypt_key(self):
         return self.toInt(self.modulus(inv_key))
 
     def decrypt(self, text):
+        """
+        >>> HillCipher(numpy.matrix([[2,5],[1,6]])).decrypt('WHXYJOLM9C6XT085LL')
+        'TESTINGHILLCIPHERR'
+        """
         self.decrypt_key = self.make_decrypt_key()
         text = self.process_text(text.upper())
         decrypted = ""
 
         for i in range(0, len(text) - self.break_key + 1, self.break_key):
             batch = text[i : i + self.break_key]
-            batch_vec = list(map(self.replaceLetters, batch))
+            batch_vec = list(map(HillCipher(self.encrypt_key).replace_letters, batch))
             batch_vec = numpy.matrix([batch_vec]).T
             batch_decrypted = self.modulus(self.decrypt_key.dot(batch_vec)).T.tolist()[
                 0
             ]
-            decrypted_batch = "".join(list(map(self.replaceNumbers, batch_decrypted)))
+            decrypted_batch = "".join(
+                list(map(HillCipher(self.encrypt_key).replace_numbers, batch_decrypted))
+            )
             decrypted += decrypted_batch
 
         return decrypted
 
 
-def main():
+if __name__ == "__main__":
```

Comment:
Doctests do not work in `if __name__ == "__main__":` so please bring back `main()`.